tests/test_resource.py

```diff
@@ -181,6 +181,21 @@ def test_patch_an_invalid_resource(self):
             Resource(connector, data={}).patch(payload={"Misc": "MyValue"})
 
 
+class TestPut:
+    def test_put_a_resource(self):
+        connector = mock.Mock(spec=Connector)
+        Resource(connector, data={
+            "@odata.id": "/redfish/v1/Systems/1"
+        }).put(payload={"Misc": "MyValue"})
```

Comment:
We are missing one more test that will test behavior when the `path` parameter is set.

Reply:
Thank you for spotting the missing test. It is now included.